### ISSUE TITLE
[4.x] Action-log: Unable to identify what a user have checked in (Article)

### ISSUE THREAD

**Issue by jiweigert** [joomla/joomla-cms#30704]
### Steps to reproduce the issue
Login as Administrator into Backend.
Go to Articles in the backend, create a new or modify an existing Article.
Save & close that article.

### Expected result
Having an entry in the Action-log which is noting:
- the user, 
- the action (check-in)
- what he have checked in (Article: "title of article (hyperlink to article)

### Actual result
Current entry in Action-log:
```User <adminuser> check in to table #__content```

### System information (as much as possible)
*Joomla! Version:*	Joomla! 3.9.21 Stable [ Amani ] 25-August-2020 15:00 GMT
*MySQL Version:*	5.7.31-1
*PHP Version:*		~~5.7.31-1~~ 7.3.22

### Additional comments
It's not critical, but without that information, a larger system with more users, which are creating articles, it's hard to see with the current entry, is responsible for checking in an article.![screen shot 2020-09-20 at 16 08 08](https://issues.joomla.org/uploads/1/d2316dbb3cceea17836c4bac536723bb.png)

**Comment by jiweigert** [comment]
Info: I removed the (linked) username in the screenshot.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30704">issues.joomla.org/tracker/joomla-cms/30704</a>.</sub>

**Comment by jiweigert** [comment]
Correction:
###System-Information
*PHP Version:*   7.3.22<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30704">issues.joomla.org/tracker/joomla-cms/30704</a>.</sub>

**Comment by sozzled** [comment]
Good idea; improves the quality of information displayed by the _Action Log - Joomla_ plugin .  Probably should be classified as a new feature request.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30704">issues.joomla.org/tracker/joomla-cms/30704</a>.</sub>

**Comment by jiweigert** [comment]
I'm fine with changing to feature request,
I don't expect that to be fixed for 3.9... unless it's not intended that the table name is shown in the Action-log.

Can someone please reassign this to *Action-log Plugin* when the current selected modules are incorrect?
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30704">issues.joomla.org/tracker/joomla-cms/30704</a>.</sub>

**Comment by brianteeman** [comment]
I assume this has been fixed elsewhere as I seee an action log entry for both article creation and articule updating

![image](https://user-images.githubusercontent.com/1296369/163710853-a6e4d309-8a26-40f0-ad10-564c4b9d1a0e.png)

**Comment by jiweigert** [comment]
Partly,

Created and updated articles are mentioned now by their title, that's fixed.
However, there is still a cryptic message left when a user checks in an
article:

"User jiweigert
<https://www.swedishsoul.com/administrator/index.php?option=com_users&task=user.edit&id=108>
performed
a check in to table #__content"

would be great if that can be fixed as well.

Kind regards.

Jörn-Ingo Weigert

Brian Teeman ***@***.***> schrieb am So., 17. Apr. 2022,
12:36:

**Comment by brianteeman** [comment]
Unlikely that could be fixed as you can performa checkout on multiple articles

**Comment by jiweigert** [comment]
Well, could be with a list and looping over the checked in article(s), but
i get what you mean.
but what coan surly be fixed is the mentioning of the mentioned table name
and give a more appropriate Naming like Article((s) depending on the type
of object checked in.
I'm sure, the internal name of the table are not interessting at all, even
for administrators.
#__content looks just ugly, that said.


Am So., 17. Apr. 2022 um 14:16 Uhr schrieb Brian Teeman <
***@***.***>:

**Comment by brianteeman** [comment]
Please update the title and  labels to J4 Issue and Feature request

**Comment by jiweigert** [comment]
how I do that via Joomla Issue Tracker?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30704">issues.joomla.org/tracker/joomla-cms/30704</a>.</sub>

**Comment by brianteeman** [comment]
Sorry that was a message for @zero-24 or any of the maintainers

**Comment by zero-24** [comment]
Done @brianteeman
